geojson_pydantic/features.py

```diff
@@ -28,14 +29,14 @@ class Config:
         use_enum_values = True
 
     @validator("geometry", pre=True, always=True)
-    def set_geometry(cls, v):
+    def set_geometry(cls, geometry: Any) -> Any:
         """set geometry from geo interface or input"""
-        if hasattr(v, "__geo_interface__"):
-            return v.__geo_interface__
-        return v
+        if hasattr(geometry, "__geo_interface__"):
+            return geometry.__geo_interface__
+        return geometry
 
     @classmethod
-    def validate(cls, value):
+    def validate(cls: Type[F], value: Any) -> F:
```

Comment:
This should go (in another PR) as discussed in https://github.com/developmentseed/geojson-pydantic/discussions/66#discussioncomment-3297335

Reply:
Yeah, I thought about that. I wasn't sure how settled that was so figured no harm in typing it and removing it in another PR. But if preferable I can do that and wait for it to be merged first.